mood_bot.py

```diff
@@ -0,0 +1,726 @@
+# mood_bot.py
+import asyncio
+import datetime
+import mood_error as me
+import os
+import re
+import shelve
+
+from discord.ext import commands
+from dotenv import load_dotenv
+
+
+load_dotenv()
+MOOD_TOKEN = os.getenv('DISCORD_TOKEN')
+
+mood_bot = commands.Bot(command_prefix='mood_')
+
+days_of_week = ('Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday')
+
+day_regex = re.compile(r'\b((mon|tues|wed(nes)?|thur(s)?|fri|sat(ur)?|sun)(day)?)|'
+                       r'(everyday|weekend(s)?|weekday(s)?)\b', re.I)
+
+day_dictionary = {'monday': 0, 'mon': 0, 'tuesday': 1, 'tues': 1, 'tue': 1, 'wednesday': 2, 'wednes': 2, 'wed': 2,
+                  'thursday': 3, 'thurs': 3, 'thur': 3, 'friday': 4, 'fri': 4, 'saturday': 5, 'satur': 5, 'sat': 5,
+                  'sunday': 6, 'sun': 6}
+
+""" These are the preset choices for mood check-ins. """
+mood_choices = ('good', 'bad', 'okay', 'meh', 'happy', 'sad', 'angry', 'upset', 'depressed', 'hungry',
+                'tired', 'bored', 'irritated', 'stressed', 'frustrated', 'ashamed', 'guilty', 'worried',
+                'lonely', 'prideful', 'confused', 'disgusted', 'surprised', 'disappointed')
+
+"""Power is for switching the auto_checker ON/OFF"""
+auto_checker_power_switch = 'OFF'

```

Comment:
This should definitely be a boolean

Reply:
I want to be able to 

`if auto_checker_power_switch:`

over 

`if auto_checker_power_switch == 'Foo'`

Reply:
Done. Switched auto_checker_power_switch to a boolean, but also turned it into a function that stores this boolean to a user-specific file so that users don't get intermingled and so that that the autochecker's status is remembered even when the bot is off.